examples/vue/e-commerce/public/index.html

```diff
@@ -20,7 +20,7 @@
       href="https://cdn.jsdelivr.net/npm/instantsearch.css@7.3.1/themes/reset-min.css"
     />
 
-    <script src="https://polyfill.io/v3/polyfill.min.js?features=default%2CArray.prototype.find%2CArray.prototype.includes%2CPromise%2CObject.assign%2CObject.entries"></script>
+    <script src="https://polyfill.io/v3/polyfill.min.js?version=3.111.0&features=default%2CArray.prototype.find%2CArray.prototype.includes%2CArray.prototype.findIndex%2CPromise%2CObject.assign%2CObject.entries"></script>
```

Comment:
Latest polyfill.io doesn't work, so we need to pin a specific version. It also seems that with this version we need to explicitly add the `Array.prototype.findIndex` polyfill.